.gitignore

```diff
@@ -4,6 +4,5 @@
 .jekyll-cache
 .sass-cache
 _site
-Gemfile.lock
```

Comment:
I suggest to add a comment with the link to the current [Jekyll recommendation](https://jekyllrb.com/tutorials/using-jekyll-with-bundler/#commit-to-source-control) for a starter `.gitignore`, and motivation for all differences, to avoid new Jekyll users wondering which to use.

Reply:
Sounds good. Added a bit of documentation here; it sounds like we should put a larger emphasis for this on the template repo? I hope users aren't directly copying our repo and/or its files (though I've noticed this pattern frequently when helping users in our Discussions).

Reply:
The updated `.gitignore` looks good!